src/main/kotlin/lotto/Lotto.kt

```diff
@@ -0,0 +1,54 @@
+package lotto
+
+class Lotto {
+    val numbers: List<Int>
+        get() = _numbers
+    private val _numbers = arrayListOf<Int>()
+
+    fun processLotto(lottoNumbers: List<Int> = shuffled()) {
+        _numbers.clear()
+        _numbers.addAll(lottoNumbers)
+    }
+
+    fun getContainLottoNumberSameCount(winningNumbers: List<Int>): Int {
+        return winningNumbers.count { isContainLottoNumber(it) }
+    }
+
+    fun isContainLottoNumber(winningNumber: Int): Boolean = _numbers.contains(winningNumber)
+
+    fun exchangeWinningMoney(sameCount: Int): Int {
+        return when (sameCount) {
+            FOURTH_WINNING_NUMBER_SAME_COUNT -> FOURTH_WINNING_MONEY
+            THIRD_WINNING_NUMBER_SAME_COUNT -> THIRD_WINNING_MONEY
+            SECOND_WINNING_NUMBER_SAME_COUNT -> SECOND_WINNING_MONEY
+            FIRST_WINNING_NUMBER_SAME_COUNT -> FIRST_WINNING_MONEY
+            else -> BLANK
+        }
+    }
+
+    private fun shuffled(): List<Int> {
+        val numbers = mutableSetOf<Int>()
+        while (numbers.size < 6) {
+            numbers.add((MIN_NUMBER..MAX_NUMBER).random())
+        }
+
+        return numbers.toList()
+    }
```

Comment:
```suggestion
    private fun shuffled(): List<Int> {
         return (MIN_NUMBER..MAX_NUMBER)
            .shuffled()
            .take(LOTTO_NUMBER_COUNT)
```
요구사항 페이지에서 준 힌트처럼 shuffled()를 사용하면 비교적 간단하게 중복되지 않은 값을 뽑아올 수 있습니다!

Reply:
아하..! shuffled 함수를 구현하라는 것으로 이해를 잘못했었네요 😂